week-1/G-strings-methods/exercise2.js

```diff
@@ -1,3 +1,10 @@
-var name = " Daniel  ";
+var name = "mariama";
+var namelength = name.length;
+console.log(
+  "my name is " + name + " and my name is " + namelength + " characters long"
+);
 
-console.log(message);
+function myFunction() {
+  var str = "     my name is mariama and my name is 7 characters long!     ";
```

Comment:
can you use name length variable instead of writing the the name characters long by yourself

Reply:
okay